lib/src/di/injector.dart

```diff
@@ -1,13 +1,18 @@
 import 'package:allthenews/src/data/appinfo/app_info_local_repository.dart';
-import 'package:allthenews/src/data/communication/api_key_local_repository.dart';
+import 'package:allthenews/src/data/communication/api/api_key_local_repository.dart';
+import 'package:allthenews/src/data/communication/api/ny_times_repositor.dart';
+import 'package:allthenews/src/data/communication/api/ny_times_repository_impl.dart';
 import 'package:allthenews/src/domain/appinfo/app_info_repository.dart';
 import 'package:allthenews/src/domain/authorization/api_key_repository.dart';
+import 'package:dio/dio.dart';
 import 'package:get_it/get_it.dart';
 
 void injectDependencies() {
   final locator = GetIt.instance;
   locator.registerSingleton<ApiKeyRepository>(ApiKeyLocalRepository());
   locator.registerSingleton<AppInfoRepository>(AppInfoLocalRepository());
+  locator.registerSingleton<Dio>(Dio());
```

Comment:
imo wstrzykiwanie dio ma sens tylko wówczas, gdy posiada już choćby jakieś interceptory, bo tak to kompletnie nic nie daje